### ISSUE TITLE
The Quarry 21:9 Fix doesn't start

### ISSUE THREAD

**Issue by Bimsstein** [RoseTheFlower/UltrawideIndex#11]
Im trying to open the .exe file and I do get the Administrator Message but no window is popping up or anything.

**Comment by RoseTheFlower** [comment]
Probably the anti-virus. Try [this](https://www.howtogeek.com/671233/how-to-add-exclusions-in-windows-defender-on-windows-10/) if you have Defender.

**Comment by Bimsstein** [comment]
tried that, still not starting. could it be related to using windows 11?

**Comment by RoseTheFlower** [comment]
I don't see how, though I'm still on Windows 10. Try to exclude a folder, not the file, then put the file in that folder.

**Comment by Bimsstein** [comment]
yeah i have both the file and the folder excluded already, its actually pretty weird that it just wont start.

**Comment by FransBouma** [comment]
Right-click the exe in file explorer -> Properties -> General tab, check the Unblock checkbox

**Comment by Bimsstein** [comment]
there is no unblock checkbox in the general tab

**Comment by maximalEbooks** [comment]
https://github.com/RoseTheFlower/UltrawideIndex/issues/12

please note that my post says that edge is deleteing them as detected virus, its fine that its closed, but its still a problem both Program:Win32/Wacapew.C!ml detected as low in C:\Users\Robert\Downloads\The.Quarry.Ultrawide.v1.2.zip->The Quarry Ultrawide v1.2.exe

and under ones are deleted as severe and being blocked on download with that Trojan:Script/Oneeva.A!ml detected in each file, once windows defender sees them they seem to either be blocked from download and deleted on try to download them to my computer under windows 11.
